shared/src/store/action-reducers/utils/calculate-treatments.spec.ts

```diff
@@ -68,12 +68,12 @@ function assertCatering(
 function addPatient(
     state: Mutable<ExerciseState>,
     pretriageStatus: PatientStatus,
+    // TODO: Use this parameter again. When it is ignored tests will fail.
```

Comment:
I think we have to investigate why there are no failing tests when this is not used. Because as far as I can tell this means that all the tests that should check whether some behavior occurs for some colors, are completely independent of the color and therefore (partly) useless.

Reply:
I thought the whole algorithm only relies on the pretriageStatus, to punish incorrect pretriage. As the pretriageStatus is set to the realStatus after 5 min or so, the treatments effectively use the realStatus then.

Reply:
Good point. I'll remove the parameter and the comment.